src/config/selectors.js

```diff
@@ -1,3 +1,4 @@
+export const TITLE_TEXT_ID = 'homeTitle';
```

Comment:
An id shouldn't be duplicated, instead create one id different for each title, and the name of the selector should reflect the string content.
```
HOME_TITLE_ID = 'homeTitle'
```